neutron-org/neutron-contracts
# feat: implemented sudo_tx_query_result()

This PR:

1. Adds an example handler for the `sudo_tx_query_result()` call;
2. Removes all code related to `query_transfer_transactions()`;
3. Renames the `interchain_tx` package to `neutron_sudo`.

HOW TO TEST THIS CODE?

You can find the instructions in the readme: https://github.com/neutron-org/neutron-contracts/blob/feat/check-tx-query-result-callback/contracts/neutron_interchain_queries/README.md
**IMPORTANT**: test with the https://github.com/neutron-org/neutron/pull/21 version of Neutron.
